build/publish/PublishDebian.targets

```diff
@@ -6,28 +6,23 @@
 
   <PropertyGroup>
     <SdkDebianUploadUrl>$(DotnetBlobRootUrl)/$(Product)/$(FullNugetVersion)/$(DistroSpecificArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)-$(Architecture)$(InstallerExtension)</SdkDebianUploadUrl>
-    <DebianUploadJsonFile>$(SdkDebianIntermediateDirectory)/package_upload.json</DebianUploadJsonFile>
-    <DebianRevisionNumber>1</DebianRevisionNumber>
-
-    <DebianUploadJsonContent>
-      {
-      "name":"$(SdkDebianPackageName)",
-      "version":"$(NugetVersion)-$(DebianRevisionNumber)",
-      "repositoryId":"$(REPO_ID)",
-      "sourceUrl": "$(SdkDebianUploadUrl)"
-      }
-    </DebianUploadJsonContent>
   </PropertyGroup>
 
-  <Target Name="PublishDebFilesToDebianRepo" Condition=" '$(IsDebianBaseDistro)' == 'True' AND '$(SkipPublishToDebianRepo)' != 'true' AND '$(IslinuxPortable)' != 'true' ">
+  <Target Name="PublishDebFilesToDebianRepo"
+          DependsOnTargets="SetupDebProps;"
+          Condition=" '$(IsDebianBaseDistro)' == 'True' AND '$(SkipPublishToDebianRepo)' != 'true' AND '$(IslinuxPortable)' != 'true' " >
     <Error Condition="'$(REPO_ID)' == ''" Text="REPO_ID must be set as an environment variable for debian publishing." />
     <Error Condition="'$(REPO_USER)' == ''" Text="REPO_USER must be set as an environment variable for debian publishing." />
     <Error Condition="'$(REPO_PASS)' == ''" Text="REPO_PASS must be set as an environment variable for debian publishing." />
```

Comment:
REPO_PASS is not an environment variable - it is set as a 'property'. The error text is misleading.

Reply:
Thanks, and fixed, now it is required as a MsBuild Property. by environment not is allowed anymore